custom_model_runner/datarobot_drum/drum/typeschema_validation.py

```diff
@@ -20,6 +20,13 @@
 T = TypeVar("T")
 
 
+def list_str(l):
```

Comment:
as this is only used in the one function, why not just move it there and get rid of the docs.

also, just return `[str(el) for el in input_list]`

Reply:
that would still have the [] in the output, which we need to get rid of.